backend/lambdas/api/repo/repo/github_util/github_utils.py

```diff
@@ -47,6 +47,11 @@ def _build_queries(req_list, service, authz):
     count = 0
     for req in req_list:
         branch_name = req.get("branch")
+
+        # Escape Double quotes in branch name. Leaving double quotes in will affect the graphql query
+        if branch_name:
+            branch_name = branch_name.replace('"', '\\"')
```

Comment:
Should this just go into the if branch_name block that's already present a few lines down?

Reply:
Yes, it should. Updating the PR now